go.mod

```diff
@@ -38,10 +38,10 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
 	github.com/hashicorp/consul-awsauth v0.0.0-20220713182709-05ac1c5c2706
 	github.com/hashicorp/consul-net-rpc v0.0.0-20221205195236-156cfab66a69
-	github.com/hashicorp/consul/api v1.26.1
-	github.com/hashicorp/consul/envoyextensions v0.5.1
+	github.com/hashicorp/consul/api v1.27.0
```

Comment:
DId you intend on these mod file changes being in this PR?

Reply:
yes, they didn't originally get merged in when I did the updates of the submodules because the release branches had already been cut